highfidelity/hifi
# Various Cellscience Fixes

This PR:

- Adds a world offset parameter, in case we decide to move all of the cellscience content closer to the ocean.
- Removes a callback that made everything ungrabbable and possible prematurely set userData for some navigation buttons which could cause them to not appear

To test: 
1. Visit hifi://cellscience and observe that things work as they did before and all 4 nav buttons appear
